app/gui/integration-test/dashboard/actions/DrivePageActions.ts

```diff
@@ -212,6 +212,7 @@ export default class DrivePageActions<Context> extends PageActions<Context> {
       dragRowToRow(from: number, to: number) {
         return self.step(`Drag drive table row #${from} to row #${to}`, async (page) => {
           const rows = locateAssetRows(page)
+          rows.nth(from).click()
```

Comment:
what does this do?

Reply:
See: `Now dragging can only be started over a selected row.`

This change selects a row to perform dragging :)